README.md

```diff
@@ -221,6 +221,7 @@ The list of arguments for `hls`-mode is presented below:
 * `--out-sv-lib <PATH>`: *optional* filesystem-path option; used to specify the output SystemVerilog file for generated operations library.
 * `--out-dfcir <PATH>`: *optional* filesystem-path option; used to specify the output DFCIR file.
 * `--out-firrtl <PATH>`: *optional* filesystem-path option; used to specify the output FIRRTL file.
+* `--out-dot <PATH>`: *optional* filesystem-path option; used to specify the output .dot file.
```

Comment:
The following renaming is proposed:
".dot" -> "DOT"

It will look more relevant to existing descriptions in this file

Reply:
Fixed.